spec/requests/api/v1/events_spec.rb

```diff
@@ -121,4 +122,85 @@
       end
     end
   end
+
+  describe 'POST /events/estimate_fees' do
+    let(:charge) { create(:standard_charge, :instant, plan:, billable_metric: metric) }
+
+    before { charge }
+
+    it 'returns a success' do
+      post_with_token(
+        organization,
+        '/api/v1/events/estimate_fees',
+        event: {
+          code: metric.code,
+          external_customer_id: customer.external_id,
+          properties: {
+            foo: 'bar',
+          },
+        },
+      )
+
+      aggregate_failures do
+        expect(response).to have_http_status(:success)
+
+        expect(json[:fees].count).to eq(1)
+
+        fee = json[:fees].first
+        expect(fee[:lago_id]).to be_nil
+        expect(fee[:lago_group_id]).to be_nil
+        expect(fee[:item][:type]).to eq('instant_charge')
+        expect(fee[:item][:code]).to eq(metric.code)
+        expect(fee[:item][:name]).to eq(metric.name)
+        expect(fee[:amount_cents]).to be_an(Integer)
+        expect(fee[:amount_currency]).to eq('EUR')
+        expect(fee[:vat_amount_cents]).to be_an(Integer)
+        expect(fee[:vat_amount_currency]).to eq('EUR')
+        expect(fee[:units]).to eq("1.0")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)